cooker/bin/commands/start/index.js

```diff
@@ -33,15 +33,40 @@ async function startCommand(options) {
 
   let app;
   let server;
+  const servers = [];
 
   // Start express app
   app = express();
 
   app.use(express.static(config.PUBLIC_DIRECTORY));
   app.get("*", requestListener);
 
-  // Create our server and socket instance
-  server = http.createServer(app);
+  const { networkInterfaces } = require("os");
+
+  const nets = networkInterfaces();
+  const netresults = Object.create(null); // Or just '{}', an empty object
+
+  for (const name of Object.keys(nets)) {
```

Comment:
Super low-priority, not-even-a-concern thing here: We could swap out this nested loop into something like:
```js
for (const [name, net] of Object.entries(nets)) {
  // the code
}
```

Reply:
I'm curious, why this change? The update doesn't really gain us anything as we'll still need to have the nested loop on the net. It will just become:

```
net.forEach((interface) => { etc... }
```

Rather than:
```
for (const net of nets[name]) { etc... }
```

Reply:
Gah, that's true. All good then! Marking as resolved. Thanks